### DISCUSSION TITLE
Pimcore Academy - Customer Management Framework - Modules and Functionality of the Customer Management Framework

### DISCUSSION THREAD

**Discussion by michaelopre** [pimcore/pimcore#15326] [Q&A / Support] [answered]
Hi,

I always fail on the first question; all other are no problem.
It is the question about the Modules and Functionality of the Customer Management Framework:

1. Full-featured CRM functionality
2. Management of Customer Data
3. Management of Customer Activities
4. Duplicates detection
5. Single Sign On with Social Logins
6. Full-featured out-of-the-box customer registration process

Let me explain what I think to understood:

1. actually yes, but I have to implement everything I need or what I think a CRM has to include -> therefore NO
2. yes, no further explanation needed -> therefore YES
3. actually yes, but I'm not sure what means "Management", Activities can be tracked, but not really managed, nevertheless -> therefore YES
4. of course -> therefor YES
5. no, because it has been removed at Version 4 of the CMF -> therefore NO
6. no, because I cannot recognize any hints for that -> therefor NO

That means in my estimation it is only 2, 3 and 4 which is correct. But it isn't.

Do you have any Idea or explanation?

Thanks in advance!

Best,
Michael

**Comment by michaelopre** [comment]
Hi @NiklasBr.

it seems that you very proficient here, please can you help me again with an advice? May be for this thread as well #15308 ?

Best,
Michael

**Reply by michaelopre** [reply, marked as answer]
I've solved the answer, after watching and listen carefully. And selected:

- Management of Customer Data
- Management of Customer Activities
- Duplicates detection
- Single Sign On with Social Logins

But I will be honest, it is very confusing, because in the architecture overview lessen is a remark below the video:

'''
With Version 4 of Customer Management Framework, SSO and Social Logins were removed.

We recommend using standard third party oauth client libraries (e.g. League/oauth2-client) for implementing SSO and Social Logins. 
'''

In my eyes does this mean, that SSO is not part of the modules and functionality anymore. Because if you consider that SSO is implementable by a third party plugin all modules that not implemented belonging to the modules and functionality.

Ok, who knows. Now it is solved.

**Reply by miro-kodet** [reply]
This issue is still apart of Pimcore Academy and I'd consider it as a bug as SSO has been removed

**Reply by fashxp** [reply]
you're right ... fixed it.
